src/platform/packages/shared/kbn-esql-utils/src/utils/query_parsing_helpers.ts

```diff
@@ -132,6 +133,43 @@ export const getTimeFieldFromESQLQuery = (esql: string) => {
   return columnName;
 };
 
+export const getSearchQueryString = (esql: string) => {
```

Comment:
@akowalska622 thanx for working on this! Can you add some tests too? It will be easier to identify cases

Reply:
Added in [7964f97](https://github.com/elastic/kibana/pull/232964/commits/7964f9750f764968fd2bf3bf0d36b8c2234a23be)